core-bundle/tests/Controller/ContentElement/TableControllerTest.php

```diff
@@ -104,7 +107,10 @@ public function testOutputsTableWithHeaderAndFooter(): void
             </div>
             HTML;
 
-        $this->assertSameHtml($expectedOutput, $response->getContent());
+        $html = $response->getContent();
+
+        $this->assertNotFalse($html);
+        $this->assertSameHtml($expectedOutput, $html);
```

Comment:
I think I’d change the `assertSameHtml()` method here instead:

```php
protected function assertSameHtml(string $expected, mixed $actual, string $message = ''): void
{
    $this->assertIsString($actual);
    $this->assertSame(
        $this->normalizeWhiteSpaces($expected),
        $this->normalizeWhiteSpaces($actual),
        $message
    );
}
```
WDYT?

Reply:
Changed in 8b52d39a40b3958570399bf26f47a1fc868f72b4.